e2etests/tests/unnumbered.go

```diff
@@ -0,0 +1,199 @@
+		ginkgo.It("session should be established and routes to be validated", func() {
+			cr := frrk8sv1beta1.FRRConfiguration{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      "native-for-" + peer.Name,
+					Namespace: k8s.FRRK8sNamespace,
+				},
+				Spec: frrk8sv1beta1.FRRConfigurationSpec{
+					BGP: frrk8sv1beta1.BGPConfig{
+						Routers: []frrk8sv1beta1.Router{
+							{
+								ASN: infra.FRRK8sASN,
+								Neighbors: []frrk8sv1beta1.Neighbor{
+									{
+										DynamicASN: frrk8sv1beta1.ExternalASNMode,
+										Interface:  "net0",
+										ToAdvertise: frrk8sv1beta1.Advertise{
+											Allowed: frrk8sv1beta1.AllowedOutPrefixes{
+												Mode: frrk8sv1beta1.AllowAll,
+											},
+										},
+									},
+								},
+								Prefixes: []string{"5.5.5.5/32", "fc00:f888:ccd:e793::1/128"},
+							},
+						},
+					},
+					NodeSelector: metav1.LabelSelector{
+						MatchLabels: map[string]string{
+							"kubernetes.io/hostname": node.GetLabels()["kubernetes.io/hostname"],
+						},
+					},
+				},
+			}
+			ginkgo.DeferCleanup(func() {
+				err := updater.CleanFRRConfiguration(cr)
+				Expect(err).NotTo(HaveOccurred())
+			})
+
+			err := updater.Update([]corev1.Secret{}, cr)
+			Expect(err).NotTo(HaveOccurred())
+
+			validate(peer, []string{"5.5.5.5", "fc00:f888:ccd:e793::1"})
+		})
+	})
+
+	// This spec is added to verify that the point-to-point setup works
+	ginkgo.Context("with raw config", func() {
+		ginkgo.It("session should be established and routes to be validated", func() {
+			raw := `
+		frr defaults traditional
+		no ipv6 forwarding
+		router bgp 65000
+			no bgp ebgp-requires-policy
+			no bgp network import-check
+			neighbor net0 interface remote-as external
+			address-family ipv6 unicast
+				network fc00:f888:ccd:e793::1/128
+				neighbor net0 activate
+			exit-address-family
+			address-family ipv4 unicast
+				network 5.5.5.5/32
+				neighbor net0 activate
+			exit-address-family
+		exit
+		end
+`
+
+			cr := frrk8sv1beta1.FRRConfiguration{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      "raw-for-" + peer.Name,
+					Namespace: k8s.FRRK8sNamespace,
+				},
+				Spec: frrk8sv1beta1.FRRConfigurationSpec{
+					NodeSelector: metav1.LabelSelector{
+						MatchLabels: map[string]string{
+							"kubernetes.io/hostname": node.GetLabels()["kubernetes.io/hostname"],
+						},
+					},
+					Raw: frrk8sv1beta1.RawConfig{
+						Config:   raw,
+						Priority: 5,
+					},
+				},
+			}
+			ginkgo.DeferCleanup(func() {
+				err := updater.CleanFRRConfiguration(cr)
+				Expect(err).NotTo(HaveOccurred())
+			})
+
+			err := updater.Update([]corev1.Secret{}, cr)
+			Expect(err).NotTo(HaveOccurred())
+
+			validate(peer, []string{"5.5.5.5", "fc00:f888:ccd:e793::1"})
+		})
+	})
+})
+
+func validate(peer *frrcontainer.FRR, prefixes []string) {
+	Eventually(func() error {
+		neighbors, err := frr.NeighborsInfo(peer)
+		Expect(err).NotTo(HaveOccurred())
+		for _, n := range neighbors {
```

Comment:
I think we should also verify that there are actual neighbors (len > 0 or something)